pkg/virtual/workspaces/registry/rest.go

```diff
@@ -163,27 +150,29 @@ func (s *REST) NamespaceScoped() bool {
 	return false
 }
 
-func (s *REST) getPrettyNameFromInternalName(user kuser.Info, internalName string) (string, error) {
+func (s *REST) getPrettyNameFromInternalName(user kuser.Info, orgName, internalName string) (string, error) {
```

Comment:
orgClusterName

Reply:
Fixed in commit c48becc80ef541475a0e032a26cfa6a9024c5ac8